### ISSUE TITLE
Group Messages and Logout

### ISSUE THREAD

**Issue by tobagin** [chrishubert/whatsapp-api#123]
When I terminate a session, it doesn't seem to logout of the session on the whatsapp, it still shows on mobile app as paired. is that right or am I doing something wrong?

also how do I fetch group messages, I haven't been able to find a way, when I use a serialized chatid(###########-##########@g.us) the fetchMessages doesn't like it and throws a 404 error. is there a way of doing it that I missed?

thanks for the awesome project!

**Comment by avoylenko** [comment]
You are right, the mobile app still shows as paired after terminating the session. It seems the issue happens only when the browser runs in headless mode, need to investigate it furtherer.

The error 404 means that group chat you are trying to get the messages for is not exists. Could you double check this?
